src/expycted/internals/utils.py

```diff
@@ -1,3 +1,5 @@
+import os
+from functools import wraps
 from typing import Callable
 
 
```

Comment:
> You would need to additionally annotate `to_not_fn` with traceback hiding for this to work properly with _negated_ methods. Otherwise it's printing part of tracebacak related to `to_not_fn`.

Did you test that functionality?

Reply:
> > You would need to additionally annotate `to_not_fn` with traceback hiding for this to work properly with _negated_ methods. Otherwise it's printing part of tracebacak related to `to_not_fn`.
> 
> @petereon Have a look at the change regarding negated assertions.

Perfect.